internal/host/validator.go

```diff
@@ -748,6 +749,8 @@ func (v *validator) printSufficientNetworkLatencyRequirementForRole(c *validatio
 		}
 		sort.Strings(hostNames)
 		return fmt.Sprintf("Network latency requirements of less or equals than %.3f ms not met for connectivity between %s and %s.", *c.clusterHostRequirements.Total.NetworkLatencyThresholdMs, c.host.RequestedHostname, strings.Join(hostNames, ","))
+	case ValidationPending:
+		return "Missing packet loss information."
```

Comment:
I guess that the string should be missing network latency information

Reply:
horrors of c&p... :cry: